### PR TITLE
Connections: Turn on feature toggle by default

### PR DESCRIPTION
**Pull request by mikkancso**
**What is this feature?**

We plan to make this feature GA with Grafana 9.5, so I'm turning on this feature toggle by default.

### CODE REVIEW

**Review by leventebalogh — APPROVED** [review]
LGTM 👍

File: pkg/services/featuremgmt/registry.go [hunk 1]
```diff
@@ -232,6 +232,7 @@ var (
 			Name:        "dataConnectionsConsole",
 			Description: "Enables a new top-level page called Connections. This page is an experiment that provides a better experience when you install and configure data sources and other plugins.",
 			State:       FeatureStateAlpha,
+			Expression:  "true", // turned on by default
```
Comment (leventebalogh) [inline pkg/services/featuremgmt/registry.go:235]: You probably also need to change the state to `FeatureStateStable`.

**Review by andresmgot — APPROVED** [review]

**Comment by andresmgot** [comment]
you need to run the generator `pkg/services/featuremgmt/toggles_gen_test.go`  to fix the errors in the CI.

**Comment by mikkancso** [comment]
Thanks!
I've rebased to main and regenerated the required files

**Comment by andresmgot** [comment]
now there are some a11y check failing. I don't think those are related to your PR, maybe rebase again or ping the team responsible of the check to know how to fix it?

**Review by chri2547 — APPROVED** [review]
Thank you for the contribution!

**Comment by mikkancso** [comment]
I increased the error threshold for pa11y, because we couldn't figure out how to fix this error. I plan to create a ticket about this for the @grafana/user-essentials team after merging this PR.

**Pull request merged**